docs/releases/v1.4.2.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+
+## Changelog
+
+Changes between `1.4.0` and this release: `1.4.1`
```

Comment:
We are changing the version of Kustomize and a big refactor that needs downtime. In my opinion this is not a minor version update, but a major, even though we are not updating nginx, we should move to 1.5.0

Reply:
We are not changing the kustomize version: https://github.com/sighupio/fury-kubernetes-ingress/pull/19#discussion_r394920680.

This is a "bugfix" to avoid code duplication

Reply:
Yes we are, the previous version happend to be kustomize v3 compatible just by luck :) And we are just now introducing the usage of v3 features.

This is not a bugfix. It's a refactor with implied downtime.

Reply:
Fair enough. Pushing a minor version ;)